themes/socialbase/src/Plugin/Preprocess/SocialBaseBlock.php

```diff
@@ -128,11 +128,15 @@ public function preprocess(array &$variables, $hook, array $info) {
       $variables['content']['search_form']['#attributes']['role'] = 'search';
       $variables['content']['search_form']['actions']['submit']['#is_button'] = FALSE;
       $variables['content']['search_form']['actions']['#addsearchicon'] = TRUE;
-      $variables['content']['search_form']['actions']['submit']['#addsearchicon'] = TRUE;
-
       if ($region == 'hero') {
         $variables['content']['search_form']['#attributes']['class'][] = 'hero-form';
         $variables['content']['search_form']['#region'] = 'hero';
+        $variables['content']['search_form']['actions']['submit']['#addsearchicon'] = TRUE;
+      }
+      elseif ($region == 'content_top') {
+        $variables['content']['search_form']['#region'] = 'content-top';
+        $variables['content']['search_form']['search_input_content']['#attributes']['placeholder'] = t('What are looking for ?');
```

Comment:
@MaikelGG, the text is incorrect. Could you change it to something like `What are you looking for?`